bridge/src/main/java/com/bitwarden/bridge/model/SharedAccountDataJson.kt

```diff
@@ -13,7 +13,7 @@ import java.time.Instant
  * @param accounts The list of shared accounts.
  */
 @Serializable
-data class SharedAccountDataJson(
+internal class SharedAccountDataJson(
```

Comment:
Can this be an internal data class?

Reply:
Totally :) Nice catch:

5643141517755254b869d585aa819c0fbbf29e8b